theme/layouts/_navigation.ejs

```diff
@@ -1,9 +1,24 @@
+<nav id="main-navigation">
+  <% function recNav(navig) { %>
+    <ul> 
+      <% for ({title, link, sub_items} of navig) { %>
+        <li>
+            <a href="<%= link %>"><%= title %></a>
+              <% if (sub_items) { %>
+                <a class="has-children" href="<%= link %>"></a>
+                  <% recNav(sub_items);
+                } %>       
+        </li>
+   <% } %>
+    </ul>
+<% }
+  recNav(site.navigation) %>
+</nav>
+
 <nav>
-  <ul>
-    <% for (const {title, link} of site.navigation) {%>
-      <li>
-        <a href="<%- link %>" target="_blank"><%- title %></a>
-      </li>
-    <% } %>
-  </ul>
+<select id="mobile-nav">
```

Comment:
Seems like the JavaScript part is yet missing.

In order to make those items navigatable, you need to listed to onchage event:
Here is [an example used in cmints.io](https://github.com/cmints/website/blob/master/theme/js/main.js)